package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "lint": "standard",
     "lint:fix": "standard --fix",
-    "redis": "docker run -p 6379:6379 --rm redis:3.0.7",
+    "redis": "docker run -p 6379:6379 --rm redis",
```

Comment:
This implies a significant jump in Redis versions. Current version is `redis:6`. I would think that this change incurs a semver major release. To avoid that, this should be changed to `redis:3`.

For the major, this should be changed to `redis:6` or `redis:6-alpine`.

Reply:
I understand and agree with you but this is already what has been used in the GitHub CI:
https://github.com/fastify/fastify-rate-limit/blob/aef5e266bc6c07918bbfbaf73e910f377fd683d9/.github/workflows/ci.yml#L23

Reply:
I think this is ok.